src/app/utilities/tests/mockData.js

```diff
@@ -181,28 +181,31 @@ const teams = [
 
 const groups = [
     {
-        id: 25,
-        name: "A new team",
-        members: ["test@test.com", "test2@test.com", "test3@test.com"],
-        path: "a_new_team_25",
-    },
-    {
-        id: 15,
-        name: "crispin",
-        members: ["admin@test.com", "data@vis.com"],
-        path: "crispin_15",
-    },
-    {
-        id: 23,
-        name: "crumpet",
-        members: [],
-        path: "crumpet_23",
-    },
-    {
-        id: 1,
-        name: "Test Team",
-        members: [],
-        path: "test_team_1",
+        creation_date: "2021-10-12T14:32:50.913Z",
+        description: "my test group description",
+        group_name: "my test group",
+        last_modified_date: "2021-10-12T14:32:50.913Z",
+        precedence: 19,
+        role_arn: null,
+        user_pool_id: "eu-west-1_Rnma9lp2q",
```

Comment:
Might be worth considering using a different `user_pool_id` rather than a real one. Having said that I am aware this is exposed elsewhere e.g. The Makefile of the dp-identity-api so just a suggestion but by no means a requirement.